c2corg_ui/static/js/contexthelp.js

```diff
@@ -0,0 +1,76 @@
+goog.provide('app.contextHelpDirective');
+goog.provide('app.ContextHelpModalController');
+
+goog.require('app');
+
+/**
+ * This directive is used to display a contextual help modal dialog.
+ *
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ */
+app.contextHelpDirective = function($uibModal, gettextCatalog) {
+  return {
+    restrict: 'A',
+    link: function(scope, element, attrs) {
+      element.append('<span class="glyphicon glyphicon-info-sign context-help-sign">');
+      element.on('click', function() {
+        $uibModal.open({
+          controller: 'AppContextHelpModalController',
+          controllerAs: '$ctrl',
+          templateUrl: '/static/partials/contexthelpmodal.html',
+          'windowClass': 'context-help-modal',
```

Comment:
I do not really know why. I've searched in ui.bootstrap externs, ``open`` function is declared as:
```
/**
 * @param {!Object} modalOptions
 * @return {!ui.bootstrap.modalInstance}
 */
ui.bootstrap.$modal.open = function(modalOptions) {};
```
Note that I am not really conformtable with closure.